node/sc/bridge_manager.go

```diff
@@ -432,20 +433,22 @@ func (bi *BridgeInfo) GetCurrentBlockNumber() (uint64, error) {
 // DecodeRLP decodes the Klaytn
 func (b *BridgeJournal) DecodeRLP(s *rlp.Stream) error {
 	var elem struct {
+		Alias         string
 		LocalAddress  common.Address
 		RemoteAddress common.Address
 		Paired        bool
 	}
 	if err := s.Decode(&elem); err != nil {
```

Comment:
Can this logic decode the legacy journal (without Alias data)?

Reply:
Thanks for pointing out what I couldn't consider. I uploaded a new commit that resolves the compatibility issue. The current one can load both legacy journal and aliased-journal. 

Say this PR is a v0.0.3. The node with v0.0.3 can read legacy and aliased journals. However, some nodes with v0.0.1 and v0.0.2 can't load all the journals created from v0.0.3 nodes. Of course, we can make legacy node load aliased-journal, but it leads to the nodes with higher version enforce generating two journal files at every insertion for legacy structure's journal and aliased-journal. So, I didn't do that because I'm not sure it's essential or not. I hope to hear your opinion.